packages/ledger/src/v2.ts

```diff
@@ -0,0 +1,288 @@
+import {
+  Chain,
+  WalletInit,
+  EIP1193Provider,
+  ProviderAccounts,
+  createEIP1193Provider
+} from '@web3-onboard/common'
+import type {
+  EthereumProvider as LedgerEthereumProvider
+} from '@ledgerhq/connect-kit-loader'
+import { isHexString, LedgerOptionsWCv2 } from './index.js'
+import { JQueryStyleEventEmitter } from 'rxjs/internal/observable/fromEvent'
```

Comment:
```suggestion
import type { JQueryStyleEventEmitter } from 'rxjs/internal/observable/fromEvent'
```
This appears to be used just as a type, so I think we can just import the type here.

Reply:
Yes, fixed. This was taken from the WalletConnect connector code, so I just wanted to maintain consistency with it.